### DISCUSSION TITLE
Adding additional nozzles to Voron

### DISCUSSION THREAD

**Discussion by lukicsl** [OrcaSlicer/OrcaSlicer#2442] [General]
A few months ago I moved from SuperSlice, which is the preferred slicer of the Voron community to Orca. SS seems not to be extensively maintained anymore. OS is in my opinion a better choice in regards to functionality and ease of use. The only negative point I see at least from my opinion, is the lack of support for different nozzle sizes. 
I am not familiar with OS internals and how to define base profiles, though I would like to contribute to the project. can anyone point me to some doku or other info on how to define these base profiles?

**Comment by raunchyneworleans** [comment]
i second this. i or want a guide to add my own or just start profiles then i then can adjust.
